samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1430,6 +1430,108 @@
           "Resource": "*"
         }]
       }
+    },
+    "DecodeSSMParameterPolicy": {
+      "Description": "Gives access to SSM key and parameter to load secrets in this account.",
+      "Parameters": {
+        "ParameterName": {
+          "Description":"The name of the secret stored in SSM in your account."
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:DescribeParameters"
+            ],
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:GetParameters",
+              "ssm:GetParameter"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${parameterName}",
+                {
+                  "parameterName": {
+                    "Ref": "ParameterName"
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "kms:Decrypt"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:ssm:${AWS::Region}:${AWS::AccountId}:alias/aws/ssm"
```

Comment:
From these docs https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-access.html

|  Each AWS account is assigned a default AWS KMS key. You can view your key by executing the following command from the AWS CLI:

I'm hesitant to grant permission to decrypt this key.

Reply:
Yes using this key is probably non-ideal, but as far as I can tell, it's what's used when you create a SecureString parameter without a defined key. I did write an alternate template for using a custom key however, as I agree this is non-ideal. 

Do you think changing the name to `DecodeSSMParameterRootKeyPolicy` and the description to indicate it grants access to the root key would be appropriate to alleviate security concerns?